### ISSUE TITLE
Mark the difference between Working Group Draft and Editors Draft

### ISSUE THREAD

**Issue by paulbastian** [openid/OpenID4VCI#125]
Suggestion: Change the title in between releases to "OpenID for Verifiable Credential Issuance - editors draft"

**Comment by tlodderstedt** [comment]
Hi, I'm not comfortable with changing the title of the document depending on the deployment target. That means we need to (in github) change the title for every WG publication to a number and then revert it back to "editors draft" afterwards. I suggest to only use the numbered scheme.

**Comment by jogu** [comment]
Maybe the GitHub pipeline can be made to add the 'editors draft' automatically for the copy published to GitHub pages?

**Comment by selfissued** [comment]
All other working groups unconditionally include the draft number in the title, as @tlodderstedt suggested.

**Comment by jogu** [comment]
I don't think it is all working groups, e.g. FAPI does not seem to: https://openid.bitbucket.io/fapi/fapi-2_0-security-profile.html

**Comment by selfissued** [comment]
I didn't know that about FAPI.  Looking at the draft, that means that the spec would need to be edited to add a version number before publishing to openid.net/specs/ (just like the current state of this spec).

**Comment by jogu** [comment]
I'd suggest closing this. We have editor's draft in the title and have published at least one new revision successfully since. I have a script that automatically updates the title to include the draft number during publishing.
